test/unit/cp4d-authenticator.test.js

```diff
@@ -31,7 +31,6 @@ const getTokenSpy = jest
 describe('CP4D Authenticator', () => {
   it('should store all CONFIG options on the class', () => {
     const authenticator = new CloudPakForDataAuthenticator(CONFIG);
-
```

Comment:
Is this newline evil?? 😆

Reply:
Not asking to change it back, just thought it was kind of funny

Reply:
Reflex